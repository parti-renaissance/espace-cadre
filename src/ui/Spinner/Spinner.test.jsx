import { render } from '@testing-library/react'
import Spinner from 'ui/Spinner/Spinner'

vi.mock('@mui/material', () => ({
  Grid: ({ children, container, item }) => (
    <div className={['mock-grid', container && 'container', item && 'item'].join(' ')}>{children}</div>
  ),
  CircularProgress: () => <div>mock-circular-progress</div>,
}))

describe('Spinner', () => {
  it('displays spinner', () => {
    const { container } = render(<Spinner />)

    expect(container).toMatchSnapshot()
  })
})
