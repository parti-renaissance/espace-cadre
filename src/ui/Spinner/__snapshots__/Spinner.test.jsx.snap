// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spinner > displays spinner 1`] = `
<div>
  <div
    class="mock-grid container "
  >
    <div
      class="mock-grid  item"
    >
      <div>
        mock-circular-progress
      </div>
    </div>
    <div
      class="mock-grid  item"
    >
      <strong>
        Page en cours de chargement
      </strong>
    </div>
  </div>
</div>
`;
